.eslintrc.cjs

```diff
@@ -13,6 +13,6 @@ module.exports = {
   plugins: ["@typescript-eslint"],
   extends: ["plugin:@typescript-eslint/recommended"],
   rules: {
-    "no-console": "error"
+    "no-console": "warn"
```

Comment:
We still have lots of console.log in the codebase so I had to switch this back to warn

Reply:
Ah, interesting. I thought I'd caught them all on main. Though I was originally going to use warn so I'm happy either way